tests/test.php

```diff
@@ -6,7 +6,7 @@
 # See PasswordHash.php for more information.
 #
 
-require 'PasswordHash.php';
+require dirname(dirname(__FILE__)) . '/src/PasswordHash.php';
```

Comment:
Instead of the double use of `dirname`, do `dirname(__FILE__) . '../src/PasswordHash.php'`